.eslintrc.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
       ],
       "max-len": [
         2,
-       80
+        150
```

Comment:
Heh, this is kind of a cop out.  You do you HAVE to extend the length of the lines to 150? Are you unable to fit in 80 columns?